samples/quickstart.js

```diff
@@ -0,0 +1,143 @@
+/**
```

Comment:
The header here doesn't match headers elsewhere.  Uses /* */ style over // style